### DISCUSSION TITLE
This auto rotation in models (stl) viewer is really stupid.

### DISCUSSION THREAD

**Discussion by Pierdek** [community/community#83073] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, 
please add somewhere button to disable auto rotation for models viewer. I've been trying to compare some models and I can't.

Regards,
Piotr

https://github.com/community/community/assets/7024201/c392aa1a-e4d4-44d0-852b-7680beb20437

**Comment by iitsalex** [comment]
10000% agree! Not only is it frustrating, it is also really brutal accessibility-wise. Spinning figures can be really rough on my eyes when I'm having a migraine.
